From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aditya Prayoga <aditya@kobol.io>
Date: Tue, 15 Sep 2020 20:04:22 +0700
Subject: arm64: dts: rockchip: helios64: label the internal USB hub

Labels for the onboard hub, its ports 1-3 and the RTL8156 so LED
triggers can reference the ports. No property changes.

Split out of add-board-helios64.patch (Aditya Prayoga, rebased onto
mainline by rpardini for 6.3); the DTB content is unchanged.

Signed-off-by: Aditya Prayoga <aditya@kobol.io>
Assisted-by: LLM
Signed-off-by: Igor Velkov <iav@iav.lv>
---
 arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts b/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
index 111111111111..222222222222 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
@@ -666,28 +666,28 @@ &usbdrd3_1 {
 		#address-cells = <1>;
 		#size-cells = <0>;
 
-		hub@1 {
+		int_hub: hub@1 {
 			compatible = "usb2109,0815";
 			reg = <1>;
 			#address-cells = <1>;
 			#size-cells = <0>;
 
-			port@1 {
+			int_hub_port1: port@1 {
 				reg = <1>;
 				#trigger-source-cells = <0>;
 			};
 
-			port@2 {
+			int_hub_port2: port@2 {
 				reg = <2>;
 				#trigger-source-cells = <0>;
 			};
 
-			port@3 {
+			int_hub_port3: port@3 {
 				reg = <3>;
 				#trigger-source-cells = <0>;
 			};
 
-			device@4 {
+			usb_lan: device@4 {
 				compatible = "usbbda,8156";
 				reg = <4>;
 				#address-cells = <2>;
-- 
Armbian

